core/translate/schema.rs

```diff
@@ -645,72 +648,72 @@ pub fn translate_drop_table(
 
     let null_reg = program.alloc_register(); //  r1
     program.emit_null(null_reg, None);
-    let tbl_name_reg = program.alloc_register(); //  r2
+    let r2_reg = program.alloc_register(); //  r2
     let table_reg = program.emit_string8_new_reg(tbl_name.name.0.clone()); //  r3
     program.mark_last_insn_constant();
     let table_type = program.emit_string8_new_reg("trigger".to_string()); //  r4
     program.mark_last_insn_constant();
     let row_id_reg = program.alloc_register(); //  r5
 
-    let table_name = "sqlite_schema";
-    let schema_table = schema.get_btree_table(table_name).unwrap();
-    let sqlite_schema_cursor_id = program.alloc_cursor_id(
-        Some(table_name.to_string()),
+    let schema_table = schema.get_btree_table(SQLITE_TABLEID).unwrap();
+    let sqlite_schema_cursor_id_0 = program.alloc_cursor_id(
+        //  cursor 0
+        Some(SQLITE_TABLEID.to_string()),
         CursorType::BTreeTable(schema_table.clone()),
     );
     program.emit_insn(Insn::OpenWrite {
-        cursor_id: sqlite_schema_cursor_id,
+        cursor_id: sqlite_schema_cursor_id_0,
         root_page: 1usize.into(),
-        name: tbl_name.name.0.clone(),
+        name: SQLITE_TABLEID.to_string(),
     });
 
     //  1. Remove all entries from the schema table related to the table we are dropping, except for triggers
     //  loop to beginning of schema table
     let end_metadata_label = program.allocate_label();
     let metadata_loop = program.allocate_label();
     program.emit_insn(Insn::Rewind {
-        cursor_id: sqlite_schema_cursor_id,
+        cursor_id: sqlite_schema_cursor_id_0,
         pc_if_empty: end_metadata_label,
     });
     program.preassign_label_to_next_insn(metadata_loop);
 
     //  start loop on schema table
     program.emit_insn(Insn::Column {
-        cursor_id: sqlite_schema_cursor_id,
+        cursor_id: sqlite_schema_cursor_id_0,
         column: 2,
-        dest: tbl_name_reg,
+        dest: r2_reg,
```

Comment:
Can you still use descriptive names for the registers? (this one and all the other `r<number>` ones)